File: Pemrograman1/Modul5/C/PRAK505-2310817310009-MuhammadFauzanAhsani.c
#include <stdio.h>

void Biodata(int birthYear, char name[], char dom[]){
    int tahun_sekarang = 2020;
    printf("Perkenalkan Nama Saya : %s\n", name);
    printf("Umur saya : %d\n", tahun_sekarang - birthYear);
    printf("Saya Adalah Angkatan : %d\n", tahun_sekarang);
    printf("Asal Saya dari : %s", dom);
}

int main(){
    int tahunLahir;
    char Namaku[20], Asal[15];

    scanf("%d\n", &tahunLahir);
    scanf("%[^\n]%*c", &Namaku);
    scanf("%[^\n]%*c", &Asal);

    Biodata(tahunLahir, Namaku, Asal);
    return 0;
}

File: Pemrograman1/Modul5/Python/PRAK505-2310817310009-MuhammadFauzanAhsani.py
def Biodata(year:int, name:str, dom:str):
    tahun_sekarang = 2023
    print(f"Perkenalkan Nama Saya : {name}")
    print(f"Umur saya : {tahun_sekarang - year}")
    print(f"Saya Adalah Angkatan : {year}")
    print(f"Asal saya dari : {dom}")
    
tahunLahir = int(input())
Namaku = input()
Asal = input()

Biodata(tahunLahir, Namaku, Asal)